Core/src/org/sleuthkit/autopsy/corecomponentinterfaces/DataResultViewer.java

```diff
@@ -90,5 +90,6 @@
      * @param contentViewer content viewer to respond to selection events from
      *                      this viewer
      */
+    @Deprecated 
```

Comment:
Why deprecate this method? 

Also, assuming we proceed to do so, the @deprecated directive should be added to the method doc.